Demo/Demo/DemoMerchantAPI.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+final class DemoMerchantAPI {
+
+    static let sharedService = DemoMerchantAPI()
+
+    private init() {}
+
+    func createOrder(orderParams: CreateOrderParams, completion: @escaping ((Result<Order, URLResponseError>) -> Void)) {
```

Comment:
Could we add a line or so of documentation just describing the process here? and so it's clear that this is not part of the SDK flow

Reply:
Happy to add documentation if folks are confused. However, I think it's pretty clear that `DemoMerchantAPI` which lives in the `Demo` app is not part of SDK logic. The separate XCode projects for Demo & PayPal help achieve this split too.

Reply:
Maybe instead at top of class, does this help?

```
/// API Client used to create and process orders on sample merchant server
final class DemoMerchantAPI: { ... }
```

Reply:
Yep that's great